### PR TITLE
Load team page through JS

### PR DESCRIPTION
**Pull request by Tanza3D**

### CODE REVIEW

**Comment by Tanza3D** [comment]
this'll be moved to the DB with octon from what i know but for now i'm keeping it hardcoded, no point to move it to db yet

File: home/api/team.php [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+        social("https://www.speedrun.com/user/MegaMix_Craft", "Speedrun.com"),
+        social("https://twitch.tv/megamix_craft", "Twitch"),
+        social("https://social.untone.uk/@MegaMix_Craft", "Mastodon")
+    ]
+);
+addMember(
+    9350342,
+    "EXtremeExploit",
+    "??home.team.role.generic.developer??",
+    "Pedrito",
```
Comment (EXtremeExploit) [inline home/api/team.php:90]: omg pedrito

File: home/api/team.php [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+addMember(
+    3357640,
+    "Electroyan",
+    "??home.team.role.generic.rankingsDataEngineer??",
+    null,
+    []
+);
+addMember(
+    10238680,
+    "chromb",
```
Comment (EXtremeExploit) [inline home/api/team.php:140]: omg chromb

File: home/api/team.php [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+    [
+        social("https://twitter.com/Mono_x_Mellow", "Twitter"),
+        social("https://www.youtube.com/channel/UC7ZW8exqZ1c6eOhyajMnyZg", "Youtube"),
+        social("https://www.twitch.tv/monoxmellow", "Twitch"),
+        social("https://discord.com/users/194863870559387648", "Discord")
+    ]
+);
+
+cleanTeam();
+header("Content-Type: application/json");
```
Comment (EXtremeExploit) [inline home/api/team.php:180]: THANK YOU YESSSSSS

File: home/api/team.php [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+        "socials" => $socials
+    ];
+}
+
+$extraHtml = [
+    "Mastodon" => "rel=\"me\""
+];
+
+
+function social($link, $name, $icon = null)
```
Comment (EXtremeExploit) [inline home/api/team.php:26]: icon is never used

**Review by EXtremeExploit — DISMISSED** [review]
all golden :+1: , just a comment but super minor stuff :D

**Comment by Tanza3D** [comment]
@EXtremeExploit fixed pls rereview

**Review by EXtremeExploit — APPROVED** [review]

**Pull request merged**